code/controllers/subsystem/air.dm

```diff
@@ -83,7 +85,8 @@ SUBSYSTEM_DEF(air)
 	msg += "RB:[rebuild_queue.len]|"
 	msg += "EP:[expansion_queue.len]|"
 	msg += "AJ:[adjacent_rebuild.len]|"
-	msg += "AT/MS:[round((cost ? active_turfs.len/cost : 0),0.1)]"
+	msg += "AT/MS:[round((cost ? active_turfs.len/cost : 0),0.1)]|"
+	msg += "HG:[hot_groups.len]"
```

Comment:
i'd put the processing of hot groups right after the hotspots, dunno if anything changes tho

Reply:
i would like it to be last cause its really low priority plus it doesnt matter much if the update sound proc just moves the sound center a little bit